lib/screens/app/app.dart

```diff
@@ -110,15 +112,17 @@ class _AppState extends State<App> with WidgetsBindingObserver {
       case AppLifecycleState.paused:
         break;
       case AppLifecycleState.resumed:
-        Provider.of<ConnectionNotifier>(context, listen: false).discoverEndpoints();
```

Comment:
is this formatting needed? How big is your IDE line wrap? 
should be at min 100

![image](https://user-images.githubusercontent.com/8041901/107174842-3c72c780-6999-11eb-950a-e7048ada4aeb.png)

Reply:
Since I'm using VSCode I suppose it has the default of 80 for the Dart extension. 
I did not notice the readjustment of lines, I will change the value.

![dart setttings](https://user-images.githubusercontent.com/42857405/107248969-7921db80-69f8-11eb-88a8-aebf295f81ea.PNG)

Reply:
Original wrap restrored. ✌️